CHANGES.rst

```diff
@@ -4,12 +4,22 @@ Change Log
 git master
 ----------
 
+New features
+''''''''''''
+
+* Added support to auto-generate Binder links for examples via ``binder``
```

Comment:
Can we write "Added experimental support", or something that marks that we are testing this feature still. Binder is still on beta right?

Reply:
Or unstable support